torch/csrc/jit/passes/onnx/fixup_onnx_controlflow.cpp

```diff
@@ -206,7 +206,9 @@ void ReplaceBlockOutputWithOptional(
   Node* opt_node = ONNXOptionalNode(opt_type, block->owningGraph());
   opt_node->insertBefore(block->return_node());
   Value* block_output = block->outputs().at(i);
-  block_output->replaceAllUsesWith(opt_node->output());
+  // replace only the last node as Optional Node only affects
```

Comment:
Suggest rephrasing, since the code is not replacing the node, but replacing the value.

Reply:
Done